packages/governance/test/swingsetTests/contractGovernor/bootstrap.js

```diff
@@ -390,7 +390,7 @@ const makeBootstrap = (argv, cb, vatPowers) => async (vats, devices) => {
 
       await E(timer).tick();
       await E(timer).tick();
-      E.when(
+      await E.when(
```

Comment:
Omitted `await` a genuine bug found by linting rule?

Reply:
This is just a test, so not a bug in what we publish. It might be a fault in how the test runs; I planned to see in the test results from CI.

But it does raise a question about the idea above of avoiding semantic changes: should they also be avoided in tests? I think not.